src/sentry/scim/endpoints/members.py

```diff
@@ -530,28 +530,26 @@ def post(self, request: Request, organization) -> Response:
         """
         update_role = False
 
-        with sentry_sdk.start_transaction(
-            name="scim.provision_member", op="scim", sampled=True
-        ) as txn:
+        with sentry_sdk.configure_scope() as scope:
```

Comment:
@sl0thentr0py Is it okay to keep all the code indented below this line?

Reply:
yep lgtm